utilities/transactions/write_prepared_transaction_test.cc

```diff
@@ -4004,7 +4005,12 @@ TEST_P(WritePreparedTransactionTest, WC_WP_WALForwardIncompatibility) {
 }  // namespace ROCKSDB_NAMESPACE
 
 int main(int argc, char** argv) {
+  ROCKSDB_NAMESPACE::port::InstallStackTraceHandler();
   ::testing::InitGoogleTest(&argc, argv);
+  if (getenv("CIRCLECI")) {
+    // Looking for backtrace on "Resource temporarily unavailable" exceptions
+    ::testing::FLAGS_gtest_catch_exceptions = false;
```

Comment:
noob question - why do we ever want it to catch exceptions? Are they used in the framework to implement ASSERT_EQ etc.?

Reply:
From quick search it looks more about treating uncaught exception consistently with any other test failure (print "FAILED" rather than crash).

Reply:
From what I've observed, you don't get the exception's message with catch_exceptions=0, and you obviously don't get results for other tests in the same executable run. I don't know if there are other caveats. IIRC @ltamasi has commented on this before.